Source/WebCore/platform/cocoa/MIMETypeRegistryCocoa.mm

```diff
@@ -78,6 +78,57 @@
     return extensionsForMIMETypeMap;
 }
 
+// Specify MIME type <-> extension mappings for type identifiers recognized by the system that are missing MIME type values.
+static const HashMap<String, String, ASCIICaseInsensitiveHash>& additionalMimeTypesMap()
+{
+    ASSERT(isMainThread());
```

Comment:
For my own education, why do we want this assertion?

Reply:
I had copied this from somewhere else with similar logic. I've removed it now, since it isn't really necessary here.